src/api/veilarbvedtaksstotte/utkast.ts

```diff
@@ -31,11 +31,11 @@ export function fetchTaOverUtkast(vedtakId: number): AxiosPromise {
 }
 
 export function lagNyttUtkast(fnr: string): AxiosPromise {
-	return axiosInstance.post(`${VEILARBVEDTAKSSTOTTE_API}/utkast`, { fnr });
+	return axiosInstance.post(`${VEILARBVEDTAKSSTOTTE_API}/v2/utkast`, { fnr });
```

Comment:
Endepunktet for henting og lagring er nå likt i og med at begge benytter `POST`. Det må vel fikses i den gitte API-appen.

Reply:
Skal nå være fikset :P